CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# This is a CODEOWNERS file
+# It specifies who is maintaining the code in this repository
+
+* @cif @schani
```

Comment:
@schani Sorry this is what I meant to propose. What would you change?

Reply:
Works for me!